packages/components/src/LoadingBar/LoadingBar.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { Text } from '../Text'
+import React from 'react'
+import styled, { keyframes } from 'styled-components'
+
+const ProgressBackground = styled.div`
+  background: ${({ theme }) => theme.colors.grey100};
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`
+
+const ProgressBar = styled.div`
+  width: 276px;
+  height: 8px;
+  margin-top: 27px;
+  border-radius: 100px;
+  opacity: 0px;
+  background: ${({ theme }) => theme.colors.grey300};
+`
+
+const ProgressAnimation = keyframes`
+  0% { min-width: 5%; }
+  1% { min-width: 20%; }
+  2% { min-width: 40%; }
+  10% { min-width: 60%; }
+  50% { min-width: 80%; }
+  100% { min-width: 90%; }
+  `
+const Progress = styled.div`
+  width: 0;
+  height: 8px;
+  gap: 0px;
+  border-radius: 100px;
+  opacity: 0px;
+  background: ${({ theme }) => theme.colors.blueDark};
+  animation: ${ProgressAnimation} 300s ease;
+`
+export const LoadingBar = () => (
+  <ProgressBackground>
+    <img
+      src="images/logo-90x90.svg"
+      alt="OpenCRVS Logo"
+      width={90}
+      height={90}
+    />
+    <ProgressBar id="appSpinner">
+      <Progress id="progress" />
```

Comment:
@jamil314 this id will clash with the #progress in index.html

the styles are fairly similar but I think safer is to not let them clash

<sub>index.html</sub>
```
width: 0;
height: 8px;
border-radius: 100px;
background: #4972bb;
animation: progressAnimation 200s ease-out forwards;
```

<sub>LoadingBar.tsx</sub>
```
  width: 0;
  height: 8px;
  gap: 0px;
  border-radius: 100px;
  opacity: 0px;
  background: ${({ theme }) => theme.colors.blueDark};
  animation: ${ProgressAnimation} 300s ease;
  ```
  
 I can fix this for you

Reply:
-`<Progress id="progress" />`
+`<div id="progress" />`
I think this will make both the progessbar to have same style

Reply:
Let's rather keep the styles separate.

Generally, local scoped styles are a better idea than global ones: treeshaking styles is easier, and it's easier to find the styles from the application. It keeps the styles and the component implementation close together. It's difficult to get the global styles to flow to the Storybook of component library.

Considering that: we can keep the React side of things the same as they are now. So only thing that we need to change is the index.html id's, either 1) adding the <style> tag inside #root (React then will override root and the styles will get overridden) or 2) having unique id's for the elements in index.html, to not clash with anything. This means changing the <div id="progress"> to something like <div id="root-app-progress-bar"> or whatever.